tf/start.sh

```diff
@@ -17,22 +18,43 @@ fi
 
 game_num=$(cat $GAMEFILE)
 game_num=$((game_num + GAMES))
-echo "Starting with training.$game_num.gz as last game in window"
+file="training.${game_num}.gz"
+
+echo "Starting with '$file' as last game in window"
+
+train() {
+  unbuffer ./train.py --cfg=$1 --output=$2 2>&1 > "$ROOT/logs/$(date +%Y%m%d-%H%M%S).log"
+  gzip -9 $2
+  mv -v $2.gz $NETDIR
+}
+
 
 while true
 do
-  if [ -f "/home/folkert/data/run1/training.${game_num}.gz" ]
+  if [ -f "$ROOT/data/$file" ]
   then
     echo ""
-    unbuffer ./train.py --cfg=$CONFIG --output=$NET 2>&1 | tee /home2/logs/$(date +%Y%m%d-%H%M%S).log
-    TSTAMP=$(date +"%Y_%m%d_%H%M_%S_%3N")
-    FILE="$NETDIR/$TSTAMP.lc0"
-    mv -v $NET $FILE
+
+    # prepare ramdisk
+    rm -rf $RAMDISK/*
+    cp -r $ROOT/split/{train,test} $RAMDISK
```

Comment:
Will it copy more and more unused games into ramdisk as more data is generated? Maybe there is some way to only copy most recent files according to window size. Or maybe train/test splitting script can do that cleanup and move old games into "archive".

Reply:
I think the split script should handle this and we should just assume ramdisk is populated.

Reply:
@mooskagh I think the split script should make sure we can do a copy like this.
@Tilps if we don't coordinate here, the data for the various networks will differ, I don't think that's desirable.

Reply:
You make a good point.  I guess rsync would be something to consider.